dipy/viz/tests/test_fvtk.py

```diff
@@ -109,14 +109,17 @@ def test_colormap():
 @npt.dec.skipif(not fvtk.have_matplotlib)
 def test_colormaps_matplotlib():
     v = np.random.random(1000)
-    for name in 'jet', 'Blues', 'Accent', 'bone':
+    for name in 'jet', 'Blues', 'bone':
```

Comment:
Sure, I agree about deprecation regardless, just suggesting testing the colormap to make sure it's correct, even if deprecated.